Gemfile

```diff
@@ -12,6 +12,8 @@ gem "puma", "~> 3.0"
 gem "sass-rails", "~> 5.0"
 # Use Uglifier as compressor for JavaScript assets
 gem "uglifier", ">= 1.3.0"
+# database management
```

Comment:
I'm slightly worried about this. The original Gemfile had no rake version - does this mean it was provided by wherever the thing's hosted? Is this going to cause a conflict?